src/server/pachyderm_test.go

```diff
@@ -8646,198 +8561,6 @@ func TestPipelineDescription(t *testing.T) {
-	require.NoError(t, c.FinishCommit(pfs.DefaultProjectName, aRepo, "master", ""))
-
-	commitb1, err := c.StartCommit(pfs.DefaultProjectName, bRepo, "master")
-	require.NoError(t, err)
-	require.NoError(t, c.PutFile(commitb1, "file", strings.NewReader("foo"), client.WithAppendPutFile()))
-	require.NoError(t, c.FinishCommit(pfs.DefaultProjectName, bRepo, "master", ""))
-
-	commitInfos, err := c.WaitCommitSetAll(commitb1.Id)
-	require.NoError(t, err)
-	require.Equal(t, 5, len(commitInfos))
-
-	commita2, err := c.StartCommit(pfs.DefaultProjectName, aRepo, "master")
-	require.NoError(t, err)
-	require.NoError(t, c.PutFile(commita2, "file", strings.NewReader("bar"), client.WithAppendPutFile()))
-	require.NoError(t, c.FinishCommit(pfs.DefaultProjectName, aRepo, "master", ""))
-
-	commitInfos, err = c.WaitCommitSetAll(commita2.Id)
-	require.NoError(t, err)
-	require.Equal(t, 5, len(commitInfos))
-
-	commitb2, err := c.StartCommit(pfs.DefaultProjectName, bRepo, "master")
-	require.NoError(t, err)
-	require.NoError(t, c.PutFile(commitb2, "file", strings.NewReader("bar"), client.WithAppendPutFile()))
-	require.NoError(t, c.FinishCommit(pfs.DefaultProjectName, bRepo, "master", ""))
-
-	commitInfos, err = c.WaitCommitSetAll(commitb2.Id)
-	require.NoError(t, err)
-	require.Equal(t, 5, len(commitInfos))
-
-	jobInfos, err := c.ListJob(pfs.DefaultProjectName, "", []*pfs.Commit{commita1}, -1, true)
-	require.NoError(t, err)
-	require.Equal(t, 2, len(jobInfos)) // a1 + nil and a1 + b1
-
-	jobInfos, err = c.ListJob(pfs.DefaultProjectName, "", []*pfs.Commit{commitb1}, -1, true)
-	require.NoError(t, err)
-	require.Equal(t, 2, len(jobInfos)) // a1 + b1 and a2 + b1
-
-	jobInfos, err = c.ListJob(pfs.DefaultProjectName, "", []*pfs.Commit{commita2}, -1, true)
-	require.NoError(t, err)
-	require.Equal(t, 2, len(jobInfos)) // a2 + b1 and a2 + b2
-
-	jobInfos, err = c.ListJob(pfs.DefaultProjectName, "", []*pfs.Commit{commitb2}, -1, true)
-	require.NoError(t, err)
-	require.Equal(t, 1, len(jobInfos)) // a2 + b2
-
-	jobInfos, err = c.ListJob(pfs.DefaultProjectName, "", []*pfs.Commit{commita1, commitb1}, -1, true)
-	require.NoError(t, err)
-	require.Equal(t, 1, len(jobInfos))
-
-	jobInfos, err = c.ListJob(pfs.DefaultProjectName, "", []*pfs.Commit{commita2, commitb1}, -1, true)
-	require.NoError(t, err)
-	require.Equal(t, 1, len(jobInfos))
-
-	jobInfos, err = c.ListJob(pfs.DefaultProjectName, "", []*pfs.Commit{commita2, commitb2}, -1, true)
-	require.NoError(t, err)
-	require.Equal(t, 1, len(jobInfos))
-
-	jobInfos, err = c.ListJob(pfs.DefaultProjectName, "", []*pfs.Commit{client.NewCommit(pfs.DefaultProjectName, aRepo, "master", ""), client.NewCommit(pfs.DefaultProjectName, bRepo, "master", "")}, -1, true)
-	require.NoError(t, err)
-	require.Equal(t, 1, len(jobInfos))
-}
-
-// TestCancelJob creates a long-running job and then kills it, testing
-// that the user process is killed.
-func TestCancelJob(t *testing.T) {
-	if testing.Short() {
```

Comment:
Where is this test skipped.  I see it's skipped with the --short flag, but otherwise  it looks like it runs too me?

Reply:
It’s not. I think perhaps I selected one too many functions?